src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,100 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+type Props = {
+  total: number,
+  perPage: number
+  currentPage: number
+  onPageChange: (page: number) => void
+  visibleItems: string[]
+};
+
+export const Pagination: React.FC<Props> = ({
+  perPage,
+  currentPage,
+  onPageChange,
+  total,
+  visibleItems,
+}) => {
+  const isFirstPage = currentPage === 1;
+  const numbersOfPage = Math.ceil(total / perPage);
+  const isLastPage = currentPage === numbersOfPage;
+
+  const listItem = [];
+
+  for (let i = 1; i <= numbersOfPage; i += 1) {
+    listItem.push(
+      <li
+        className={classNames(
+          'page-item',
+          { active: currentPage === i },
+        )}
+        key={i}
+      >
+        <a
+          data-cy="pageLink"
+          className="page-link"
+          href={`#${i}`}
+          onClick={() => {
+            if (currentPage !== i) {
+              onPageChange(i);
+            }
+          }}
+        >
+          {i}
+        </a>
+      </li>,
+    );
+  }
```

Comment:
you have numbers of pages, on my opinion you can just use helper getNumbers function to make an array of pages count, and then just map through it